docs/userguide/forecasting_overview.md

```diff
@@ -40,6 +40,54 @@ Furthermore, we define the following types of time series consumed by the models
 * **Target series:** the series that we are interested in forecasting.
 * **Covariate series:** some other series that we are not interested in forecasting, but that can provide valuable inputs to the forecasting model.
 
+## Reproducibility
+
+If the user wishes to save a particular model and use it elsewhere or at a later point in time, darts leverages pickle to achieve that.
+All forecasting models support saving the model into a pickle object, by calling the `save()` function, which saves that particular ForecastingModel object instance. When the model is to be used again, the method `load()` can be used. Please note that the methods `save_model()` and `load_model()` are deprecated.
+
+**Example:**
+```python
+from darts.models import RegressionModel
+
+model = RegressionModel(lags=4)
+
+model.save("my_model.pkl")
+model_loaded = RegressionModel.load("my_model.pkl")
+
+from darts.models import NaiveSeasonal
```

Comment:
Do we need this second example? I'd say maybe the one above is enough. Also the one below seems to be saving to a `.py` file?